coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Products/ProductResponse.cs

```diff
@@ -84,5 +89,13 @@ public class ProductResponse
         /// <example> Manager, Board </example>
         [Required]
         public IEnumerable<UserGroup> AllowedUserGroups { get; set; } = new List<UserGroup>();
```

Comment:
I suggest not to add a default value despite the compiler warning. When we move to .NET 8, we will introduce the required property which I believe is a better solution when we do want to it to have a default value. This also applies for MenuItems

Reply:
Addressed in 3dee1ab727bad46b91e42622e3b19b97b7b92bf8